src/main/java/com/br/zup/gerenciadordeguildas/repositories/MembroRepository.java

```diff
@@ -9,4 +9,5 @@
     Optional<Membro> findByNome(String nome);
     Optional<Membro> findByEmail(String email);
     boolean existsByEmail(String email);
+    Iterable<Membro> findAllByRepresentanteIs(boolean booleano);
```

Comment:
Retirar o parâmetro, pois não há necessidade do mesmo.

Obs: No nome do método, a palavra is vem antes de represetante.